modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/ContentDashboardItemFactoryTracker.java

```diff
@@ -137,7 +139,7 @@ private void _addAssetVocabulary(
 	}
 
 	private void _addClassNameIdToVocabularyIfNotExist(
-		long classNameId, String name) {
+		long classNameId, int visibilityType, String name) {
```

Comment:
Should we use the `visibilityType` param in line 155 too?

Reply:
Yes, Hawkeye